libs/pilight/core/common.c

```diff
@@ -234,11 +234,11 @@ int isrunning(const char *program) {
 		return -1;
 	}
 	int i = 0;
-	char name[255], *p = name;
-	memset(&name, '\0', 255);
+	char name[1023], *p = name;
+	memset(&name, '\0', 1023);
 
 	for(i=0;i<psutil_max_pid();i++) {
-		if(psutil_proc_name(i, &p) == 0) {
+		if(psutil_proc_name(i, &p, 1024) == 0) {
```

Comment:
I would say `sizeof(name)`. And there are also two other platforms in psutil.

Reply:
Can't spot where to actually use the new parameter in the windows and freebsd code tho

Reply:
Then the functions should still match.